### PR TITLE
Merge initial environment details.

### PR DESCRIPTION
**Pull request by sMyn42**
@gwaygenomics 
/cc @yuenler

This is the first pull request I've done from a fork in a while. This is done correctly, right?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.ipynb_checkpoints/
+__pycache__/
+.Rproj.user
+.RData
+.Rhistory
+.Ruserdata
```
Comment (gwaybio) [inline .gitignore]:
```suggestion
.Ruserdata

```

Note the ["github suggestion"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes). i.e. you can directly commit the suggestion without having to do anything other than click a button.

All the suggestion is doing is to get rid of the pesky red symbol in line 6

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Examining Batch Effects on Cell Painting Data
+
+This project examines the presence of batch effects in (high-throughput) single-cell morphology data of different strains of cells subjected to various perturbations.
```
Comment (gwaybio) [inline README.md]:
```suggestion
This project examines the presence of batch effects in (high-throughput) single-cell morphology data.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+In addition to evaluating the benefits of the kBET detection algorithm, Harmony is used to correct for the observed batch effect. Ultimately, the goal is to benchmark these two processes and evaluate their effectiveness on the LINCS cell morphology data.
```
Comment (gwaybio) [inline README.md]:
```suggestion
We use [kBET](https://github.com/theislab/kBET) to first detect the extent of batch effects, and then [Harmony](https://github.com/immunogenomics/harmony) to adjust for them.
Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness single cell Cell Painting data.
```

Looks great, just some minor tweaks. Note the one-sentence-per-line rule of thumb for markdown styling (it makes editing sentences so much easier, line-by-line.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+@TODO update README
+
```
Comment (gwaybio) [inline README.md]:
```suggestion
```

No need for this, let's just delete

File: README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+We use [conda](https://docs.conda.io/en/latest/) to manage the computational environment.
+
+After installing conda, execute the following to install and navigate to the environment:
+
+```bash
+# First, install the conda environment
+conda env create -f environment.yml
+
+# Then, activate the environment and you're all set!
+conda activate myenv
```
Comment (gwaybio) [inline README.md:18]: we should name this something more specific. How about `cp-batch-effect`? Feel free to choose something else, it just shouldn't be nondescript.

File: environment.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+name: myenv
```
Comment (gwaybio) [inline environment.yml:1]: see name comment above

**Review by gwaybio — CHANGES_REQUESTED** [review]
Congrats on your first pull request in the lab! Only a couple comments/suggestions.

**Comment by gwaybio** [comment]
cc @yuenler for a good example of a pull request and pull request review

File: README.md (hunk 4 above)
Comment (sMyn42) [reply README.md]: Oops, I meant to delete that, just wanted to see is annotations were a thing in Github like in Java.

File: environment.yml (hunk 6 above)
Comment (sMyn42) [reply environment.yml:1]: Just changing this in github won't cause any merge conflicts, right?

Comment (sMyn42) [reply environment.yml:1]: I've built the environment, and I'll change it here, but I'm gonna need to put it in the .gitignore, until I clone, delete, prune, and update.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Examining Batch Effects on Cell Painting Data
+
+This project examines the presence of batch effects in (high-throughput) single-cell morphology data.
+We use [kBET](https://github.com/theislab/kBET) to first detect the extent of batch effects, and then [Harmony](https://github.com/immunogenomics/harmony) to adjust for them.
+Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness single cell Cell Painting data.
```
Comment (gwaybio) [inline README.md]:
```suggestion
Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness in single cell Cell Painting data.
```

**Review by gwaybio — APPROVED** [review]
after committing one minor suggestion, PR LGTM!

**Pull request merged**